src/kvstore/RocksEngine.h

```diff
@@ -54,10 +59,25 @@ class RocksRangeIter : public KVIterator {
     return folly::StringPiece(iter_->value().data(), iter_->value().size());
   }
 
+  rocksdb::Slice* iterateUpperBound() const {
+    return iterateUpperBound_.get();
+  }
+
+  void setIterator(rocksdb::Iterator* iter) {
```

Comment:
just `reset` will be more concision? since the class itself is an Iter, need not repeat it in the function.